src/examples/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@angular/material-examples",
```

Comment:
Let's make this `angular-material-examples` since it's not something we should ever publish under `@angular`